CHANGELOG

```diff
@@ -1,6 +1,8 @@
 Qtile x.xx.x, released xxxx-xx-xx:
     * features
       - consolidate development infrastructure on uv
+      - introduce BackgroundPoll with a new `async def apoll()` method for
```

Comment:
This is more than "introduce", isn't it? You've replaced the `ThreadPoolText` which may be a breaking change for users who've created their own widgets.

Reply:
It's not a breaking change - I wrote a migration, which will presumably autoconvert people's stuff. But yes, I can update the language here.

Reply:
How does it read now?